install/agones/Chart.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: v1
```

Comment:
Does the directory have to be `/install/agones` or could it be `/install/chart` ? (Not sure if helm needs it to be "agones" so we can do `helm install agones`)

`/install/agones/` feels kinda redundant to me, and doesn't actually explain what is contained within.

Since we have 2 install paths (yaml and chart), should it be `/install/yaml' (with the install.yaml in it) and '/install/chart` ?

Alternative option: rather than `/install/chart` /install/helm` - that's also quite explicit.

WDYT?

Reply:
If we want to execute the command `helm install agones`, the `Chart.yaml` file needs to live in the `agones` directory. In this case this folder is in `install/`.

I feel you're right, `install/agones` sound redundant, and doesn't explain what is contained in. 

For me is a good idea to create 2 paths:
* For the yaml file: `install/yaml`, and move the `install.yaml` file to it
* For the chart: For me is a good idea name it `install/helm`, and put inside it the `README.md` that actually is inside the Agones folder. The result can be:

```
agones/
├── install/
│   ├── yaml/
|   |   ├── install.yaml
|   ├── helm
|   |   ├── README.md
|   |   ├── agones/
|   |   |   ├── Chart.yaml
|   |   |   ├── values.yaml
|   |   |   ├── ...
```
Using this directory structure, to install Agones:

```bash
$ cd install/helm
$ helm install agones
```

Reply:
I like this a lot. SGTM! :+1:

Reply:
I like it too